airflow/jobs/__init__.py

```diff
@@ -23,4 +23,4 @@
 from airflow.jobs.backfill_job import BackfillJob  # noqa: F401
 from airflow.jobs.base_job import BaseJob  # noqa: F401
 from airflow.jobs.local_task_job import LocalTaskJob  # noqa: F401
-from airflow.jobs.scheduler_job import DagFileProcessor, SchedulerJob  # noqa: F401
+from airflow.jobs.scheduler_job import DagFileProcessor, DagFileProcessorRunner, SchedulerJob  # noqa: F401
```

Comment:
I would say we shouldn't export this here -- it's not a job and not needed for any of the rest of the jobs.

Reply:
Ok. Agree. I will remove it.